scripts/global.js

```diff
@@ -2408,6 +2422,8 @@ async function refreshMasternodeData(cMasternode, fAlert = false) {
                 doms.domMnTextErrors.innerHTML = ALERTS.MN_START_FAILED;
                 createAlert('warning', ALERTS.MN_START_FAILED, 6000);
             }
+        } else {
+            createAlert('warning', 'You must disable the view only mode', 6000);
```

Comment:
This would probably start spamming notifications (or at least a notif every XX seconds) because `refreshMasternodeData()` is called frequently to render the UI, no?

Also, i18n system.

Reply:
I think you are right, that alert should be moved somewhere else (the problem is that if you start a masternode in view only mode it will fail and it does not notify the user to disable it). Anyways it is out of scope for this PR so I will remove it